### PR TITLE
Prepare release notes for the v1.0.0-rc3 release

### PR DESCRIPTION
**Pull request by llucax**
Remove notes from the rc2 release and update release notes for the v1.0.0-rc3 release.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,85 +2,58 @@
...
+- `microgrid` package
 
-- `BatteryPool`'s control methods
+  * `grid`: The grid connection is now exposed as `microgrid.grid()`. This is more consistent with other objects exposed in the `microgrid` module, such as `microgrid.battery_pool()` and `microgrid.logical_meter()`.
 
-  * Original methods `{set_power/charge/discharge}` are now replaced by `propose_{power/charge/discharge}`
-  * The `propose_*` methods send power proposals to the `PowerManagingActor`, where it can be overridden by proposals from other actors.
-  * They no longer have the `adjust_power` flag, because the `PowerManagingActor` will always adjust power to fit within the available bounds.
-  * They no longer have a `include_broken_batteries` parameter.  The feature has been removed.
+  * `BatteryPool`: The `include_broken_batteries` argument was removed from the control methods. This option is no longer supported.
```
Comment (daniel-zullo-frequenz) [inline RELEASE_NOTES.md]: I think this belongs to `- BatteryPool's control methods` which has been removed and not to `microgrid package`

Comment (llucax) [reply RELEASE_NOTES.md]: You mean because the `BateryPool` object is defined in `timeseries`?

Comment (daniel-zullo-frequenz) [reply RELEASE_NOTES.md]: Nevermind, I've just understood the changes done

**Review by daniel-zullo-frequenz — DISMISSED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (llucax) [reply RELEASE_NOTES.md]: Ouch, please don't approve if you see any issues and auto-merge is enabled :laughing:

Comment (llucax) [reply RELEASE_NOTES.md]: I looked at the commit and see this is not exposed via `microgrid.battery_pool()`, so OK, will put it outside the `microgrid` package.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,85 +2,58 @@
...
+    For example:
 
-- An actor that is crashing will no longer instantly restart but induce an artificial delay to avoid potential spam-restarting.
+    ```python
+    # Old:
+    pv_consumption = logical_meter.pv_consumption_power()
+    production = (logical_meter.pv_production_power() + logical_meter.chp_production_power()).build()
+    # New:
+    pv_consumption = logical_meter.pv_power().consumption().build()
+    production = (logical_meter.pv_power().production() + logical_meter.chp_power().production()).build()
```
Comment (llucax) [inline RELEASE_NOTES.md:35]: Can someone double-check this example is OK? @matthias-wende-frequenz maybe?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -2,85 +2,58 @@
...
+- The configuration flag `resend_latest` can now be changed for channels owned by the `ChannelRegistry`.
 
-- The PowerDistributingActor now supports n:m relations between inverters and batteries.
+- New formula operators for calculating `consumption()` and `production()` were added.
 
-  This means that one or more inverters can be connected to one or more batteries.
+- The `ConnectionManager` now fetches microgrid metadata when connecting to the microgrid and exposes `microgrid_id` and `location` properties of the connected microgrid.
 
-- A `PowerManagingActor` implementation.
+  Users can access this information using `microgrid.connection_manager.get().microgrid_id` and `microgrid.connection_manager.get().location`.
```
Comment (llucax) [inline RELEASE_NOTES.md:46]: Also I think this got really hard to access, I wonder if we should provide something less cryptic, like `microgrid.microgrid_id()` and `microgrid.location()`. If we agree on it I will create a new issue.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -2,85 +2,58 @@
...
+- `actor` package
 
-- `microgrid.grid()`
+  * Actors are now restarted after a small delay when they crash to avoid a busy loop and spamming the logs if the actor keeps failing to start.
 
-  * Similar to `microgrid.battery_pool()`, the Grid is now similarily accessed.
+- `microgrid` package
 
-- `BatteryPool`'s control methods
+  * `grid`: The grid connection is now exposed as `microgrid.grid()`. This is more consistent with other objects exposed in the `microgrid` module, such as `microgrid.battery_pool()` and `microgrid.logical_meter()`.
```
Comment (daniel-zullo-frequenz) [inline RELEASE_NOTES.md:19]: If you'll update this patch, I'd suggest to use `grid()` to be consistent with `logical_metter()`

```suggestion
  * `grid()`: The grid connection is now exposed as `microgrid.grid()`. This is more consistent with other objects exposed in the `microgrid` module, such as `microgrid.battery_pool()` and `microgrid.logical_meter()`.
```

File: RELEASE_NOTES.md (hunk 1 above)
Comment (llucax) [reply RELEASE_NOTES.md]: OK, this was very hard to track what was really changed and which parts were affected. @frequenz-floss/python-sdk-team when writing release notes we should think more from the user's perspective at the time to upgrade to write good notes, useful notes.

Comment (llucax) [reply RELEASE_NOTES.md]: Updated to hopefully really reflect the user-visible changes and in a useful way.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (daniel-zullo-frequenz) [reply RELEASE_NOTES.md:46]: My only minor concern is those methods might return `None` which is not very suitable for the `microgrid` high level interface. Anyway I agree currently it is hard to access it and  we can create the issue and discuss it there

File: RELEASE_NOTES.md (hunk 4 above)
Comment (llucax) [reply RELEASE_NOTES.md:19]: I intentionally used the old name here, because it was "renamed".

File: RELEASE_NOTES.md (hunk 2 above)
Comment (shsms) [reply RELEASE_NOTES.md:35]: this looks correct.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (shsms) [reply RELEASE_NOTES.md]: not sure what the confusion is,  the flag was part of the battery pools control methods,  and they've gone away: https://frequenz-floss.github.io/frequenz-sdk-python/v0.25/reference/frequenz/sdk/timeseries/battery_pool/#frequenz.sdk.timeseries.battery_pool.BatteryPool.charge

this was not about power distributing actor internals.

**Review by daniel-zullo-frequenz — APPROVED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (llucax) [reply RELEASE_NOTES.md]: It affects the access via `microgrid.battery_pool()` (which IMHO is the main thing the user cares about) and also some other public (although probably not used by users really) interfaces.

**Pull request merged**